dev/Scroller/TestUI/Scroller_TestUI.projitems

```diff
@@ -70,6 +73,10 @@
       <SubType>Designer</SubType>
       <Generator>MSBuild:Compile</Generator>
     </Page>
+    <Page Include="$(MSBuildThisFileDirectory)ScrollerMousePanningPage.xaml">
+      <SubType>Designer</SubType>
+      <Generator>MSBuild:Compile</Generator>
```

Comment:
Should you have this? `<IncludeInWindowsAppx>false</IncludeInWindowsAppx>`

Reply:
Thanks.  Fixed.  This is what happens when you just use VS to add a new page.